README.md

```diff
@@ -1 +1,160 @@
-# Rotations
+# Rotations.jl
+
+This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
+
+This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
+
+### Rotation Parameterizations
+
+1. **Rotation Matrix** `RotMatrix{T <: AbstractFloat}`
+
+    A 3 x 3 rotation matrix storing the rotation.  This is a typealias of a [FixedSizeArrays](https://github.com/SimonDanisch/FixedSizeArrays.jl) `Mat{3,3,T}`.  A rotation matrix `R` should have the property `I = R * R<sup>T</sup>`.
```

Comment:
That sucks! I've gone with a ' instead.

On 9 May 2016 at 13:16, Chris Foster <notifications@github.com> wrote:

> In README.md
> <https://github.com/FugroRoames/Rotations.jl/pull/2#discussion_r62446996>:
>
> > @@ -1 +1,160 @@
> > -# Rotations
> > +# Rotations.jl
> > +
> > +This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
> > +
> > +This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
> > +
> > +### Rotation Parameterizations
> > +
> > +1. **Rotation Matrix** `RotMatrix{T <: AbstractFloat}`
> > +
> > +    A 3 x 3 rotation matrix storing the rotation.  This is a typealias of a [FixedSizeArrays](https://github.com/SimonDanisch/FixedSizeArrays.jl) `Mat{3,3,T}`.  A rotation matrix `R` should have the property `I = R * R<sup>T</sup>`.
>
> Markdown issue - <sup> has to gooutside backtick blocks
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/2/files/fb3a7882257b93afed97cfaa9bf56669bbf410dc#r62446996>
>